stable/appmesh-controller/values.yaml

```diff
@@ -9,13 +9,13 @@ preview: false
 
 image:
   repository: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller
-  tag: v1.4.0
+  tag: v1.4.1
   pullPolicy: IfNotPresent
 
 sidecar:
   image:
     repository: 840364872350.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-envoy
-    tag: v1.17.2.0-prod
+    tag: v1.16.1.1-prod
```

Comment:
why we are decreasing the version?

Reply:
The v1.17.x doesnt work with spire. Appmesh team is working on fixing the envoy config. So for this patch release we are reverting it.